### ISSUE TITLE
Release v2.8.2

### ISSUE THREAD

**Issue by JorTurFer** [kedacore/keda#4132]
Due to the reduced k8s versions supported for KEDA v2.9, we have decided to do an exceptional release for v2.8, back porting all the fixes and CVEs we have already solved.

This issue will track all the needed PR for being back ported:

| Merged at			  | Number | SHA									 | Back ported? |
|----------------------|------|------------------------------------------|----|
| 2022-09-27T06:11:08Z | #3693 | 4c19b384ba4b4e25ab05bcf599316a36ae865b1a |  Yes  |
| 2022-10-10T11:13:44Z | #3694 | 322ff425f2ab64b437b081870e33febc7d929497 |  Yes  |
| 2022-11-04T14:58:08Z | #3762 | 7af095399587e5266cf5b0a6f5bae027c7f40f96 |  Yes  |
| 2022-11-09T07:51:30Z | #3809 | 971ab94858880d7f9b0c4ae9a38d757e640b2618 |  Yes  |
| 2022-12-02T08:06:05Z | #3946 | a489ca7cd18c8e99067587ce712287736821b83f |  Yes  |
| 2022-12-05T17:22:48Z | #3788 | d5f33499cf33cf7225442ba9f48dd4a10856b6d9 |  Yes  |
| 2022-12-07T14:20:21Z | #3599 | 5953f7b0a0616583afd94b9a376596a59fac4192 |  Yes  |
| 2022-12-08T16:47:41Z | #3954 | 455eab32135d95668305e07957c79580471b2c50 |  Yes  |
| 2022-12-09T10:42:37Z | #3989 | 3e40fc6533328ecc2a23c8905694cc44d9c237a3 |  Yes  |
| 2022-12-15T09:38:52Z | #4012 | 06f6f7d444a1ea6ed66b350bdad5be20f319462c |  WONT FIX  |
| 2022-12-16T10:18:26Z | #4006 | 6c49e261865dbcddc00c2e37e908628b2b9e7c42 |  WONT FIX  |
| 2022-12-24T11:56:14Z | #4039 | b7607fcc1809d066cb280261994481455fefaf9b |  WONT FIX  |

Additionally, we need to solve this issue during the release, fixing the v2.8 helm chart

- [ ] https://github.com/kedacore/keda/issues/4126

**Comment by pedro-stanaka** [comment]
Is it okay if we backport all of those in one PR? I started cherrypicking into one branch, but I am wondering if it is okay.

**Comment by pedro-stanaka** [comment]
PRs ordered by mergedAt.

|  Merged At   |    PR number    |
|-------------|----------------|
|  2022-09-27T06:11:08Z |  3693 |
|  2022-10-10T11:13:44Z |  3694 |
|  2022-11-04T14:58:08Z |  3762 |
|  2022-11-09T07:51:30Z |  3809 |
|  2022-12-02T08:06:05Z |  3946 |
|  2022-12-05T17:22:48Z |  3788 |
|  2022-12-07T14:20:21Z |  3599 |
|  2022-12-08T16:47:41Z |  3954 |
|  2022-12-09T10:42:37Z |  3989 |
|  2022-12-15T09:38:52Z |  4012 |
|  2022-12-16T10:18:26Z |  4006 |
|  2022-12-24T11:56:14Z |  4039 |

**Comment by JorTurFer** [comment]
I think it's not a problem

**Comment by pedro-stanaka** [comment]
I opened #4135 where I am back porting all PRs except the three last ones from the table above. #4012 is specially tricky because it is using a lot of new code from APIs that I suppose was refactored. So since I am not 100% familiar with the code base I feel not safe to cherrypick. I will work on the issues I see already in #4135 and will try to make CI work. If you or another maintainer could help, would be nice. Thx.

**Comment by JorTurFer** [comment]
No worries, if you have cherry-picked the others, we can fix those 3. You have already helped a lot, with the amount of commits you have backported 🙇

**Comment by zroubalik** [comment]
Hi, #4012 is not needed, the problem it is fixing has been introduced in 2.9.0.

I also think that we don't have to do #4006 - it is rather a change in funcitonality

**Comment by JorTurFer** [comment]
> Hi, #4012 is not needed, the problem it is fixing has been introduced in 2.9.0.

perfect!

> I also think that we don't have to do #4006 - it is rather a change in funcitonality
It is indeed, but it's for solving a critical security issue (this was detected during the audit), IMO we should back port it

**Comment by zroubalik** [comment]
okay, let's proceed with #4006 in a separate PR once this one is merged

**Comment by JorTurFer** [comment]
Yeah, I'll back port that PR, I did the original PR so I hope it'll be easy :)

**Comment by JorTurFer** [comment]
KEDA v2.8.2 has been released 🥳 🥳 🥳 
@pedro-stanaka , you can pull the chart v2.8.3 when you want (and let's us know if there is any error with it)
